frontend/pages/schemas/migrations/[schemaMigration].tsx

```diff
@@ -0,0 +1,147 @@
+import { ArrowBack } from '@mui/icons-material'
+import { Button, Container, Paper } from '@mui/material'
+import { useGetSchema } from 'actions/schema'
+import { putSchemaMigration, useGetSchemaMigrations } from 'actions/schemaMigration'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
+import Loading from 'src/common/Loading'
+import ErrorWrapper from 'src/errors/ErrorWrapper'
+import Link from 'src/Link'
+import SchemaMigrator from 'src/schemas/SchemaMigrator'
+import { CombinedSchema, QuestionMigration } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+import { getStepsFromSchema } from 'utils/formUtils'
+export default function SchemaMigrationEditor() {
+  const router = useRouter()
+  const { schemaMigration: schemaMigrationId }: { schemaMigration?: string } = router.query
+  const { schemaMigrations, isSchemaMigrationsLoading, isSchemaMigrationsError } =
+    useGetSchemaMigrations(schemaMigrationId)
```

Comment:
This end point is designed to get all schema migrations, I think we need to make a new one for getting a specific migration plan.